### PR TITLE
调整部分细节

### PR DESCRIPTION
**Pull request by Initial-heart-1**
调整了部分细节
Allow edits and access to secrets by maintainers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,48 +87,48 @@ liveAtAll: false
...
+按`F5`刷新页面，按下图复制 Cookie   
 <img src="docs/img/cookie.png" width="500">
 
 </details>
 
-## 使用
-插件命令依赖 [在聊天环境执行指令](https://github.com/project-mirai/chat-command) 请确保有此插件    
+## 使用帮助
+插件命令依赖 [chat-command](https://github.com/project-mirai/chat-command) 来正确处理聊天环境下的命令，请确保有此插件，否则插件可能无法正常工作    
 如果你不会给权限可以点击下面展开查看简略教程
```
Comment (Colter23) [inline README.md:125]: 把插件的权限ID在details外面也写一遍吧
|  权限名   |  ID  |
|  ----  | ----  |
| 基础权限ID  | top.colter.bilibili-dynamic-mirai-plugin:* |
| 动态指令ID  | top.colter.bilibili-dynamic-mirai-plugin:command.bili |

File: README.md [hunk 2]
```diff
@@ -134,19 +137,19 @@ baiduTranslate:
...
+/bili <set 设置> <uid> [群/Q号]       #为目标 [群/Q号] 设置订阅内容
+/bili <color 颜色> <uid> <16进制颜色>  #为目标 UID 设置图片推送主题色
 
 [群/Q号] 不填的话默认对话所在地
 <16进制颜色> 必须带#号 例: #fde8ed
 
 下面是一些栗子:
-/bili add 487550002    :为本群/好友添加uid为487550002的订阅
-/bili add 487550002 111111  :为111111群/好友添加订阅
+/bili add 487550002    :为本群/好友添加 uid 为 487550002 的订阅
```
Comment (Colter23) [inline README.md:151]: 为**当前**群/好友...

File: README.md [hunk 3]
```diff
@@ -1,78 +1,81 @@
 # BilibiliDynamic MiraiPlugin
 
-> 一个把B站动态转发到Q群的 [Mirai](https://github.com/mamoe/mirai) 插件
+一个可低延迟检测 B 站动态并转发到 Q 群的 [Mirai](https://github.com/mamoe/mirai) 插件
```
Comment (Colter23) [inline README.md:3]: 动态/直播

Comment (Initial-heart-1) [reply README.md:3]: note: "Allow edits and access to secrets by maintainers" is checked

File: README.md [hunk 4]
```diff
@@ -1,78 +1,81 @@
...
 
-## 简介    
-低延迟检测B站动态(最低10s检测所有动态 无论多少订阅)    
-使用Java原生Api绘图  
+## 特性    
+**无论多少订阅均可在最低 10s 内检测所有动态**
```
[the remark this answers is not in the document]
Comment (Initial-heart-1) [reply README.md:6]: 但是仔细想想，避免了中文间使用空格且表意准确

File: README.md [hunk 5]
```diff
@@ -1,78 +1,81 @@
...
+### 下载
 下载插件本体: [releases](https://github.com/Colter23/bilibili-dynamic-mirai-plugin/releases)    
-插件命令依赖 [在聊天环境执行指令](https://github.com/project-mirai/chat-command) 请确保有此插件
+插件命令依赖 [chat-command](https://github.com/project-mirai/chat-command) 请确保有此插件
 
-## 配置
+### 配置
   下面是一个配置文件模板   
-  配置文件位于 根目录/config/BilibiliDynamic/BiliPluginConfig.yml  
+  配置文件位于 `mirai 根目录/config/BilibiliDynamic/BiliPluginConfig.yml`
```
[the remark this answers is not in the document]
Comment (Initial-heart-1) [reply README.md:41]: 这不是你的锅吗，我拿你的改的

File: README.md (hunk 1 above)
Comment (Initial-heart-1) [reply README.md:125]: note: "Allow edits and access to secrets by maintainers" is checked

File: README.md (hunk 2 above)
Comment (Initial-heart-1) [reply README.md:151]: note: "Allow edits and access to secrets by maintainers" is checked

**Pull request merged**